cli/tools/registry/module_content.rs

```diff
@@ -102,4 +140,269 @@ impl ModuleContentProvider {
 
     Ok(rewritten_text.into_bytes())
   }
+
+  fn add_jsx_text_changes(
+    &self,
+    specifier: &Url,
+    parsed_source: &ParsedSource,
+    text_info: &SourceTextInfo,
+    module_info: &deno_graph::ModuleInfo,
+    diagnostic_reporter: &mut dyn FnMut(SpecifierUnfurlerDiagnostic),
+    text_changes: &mut Vec<TextChange>,
+  ) -> Result<(), AnyError> {
+    static JSX_RUNTIME_RE: Lazy<regex::Regex> =
+      lazy_regex::lazy_regex!(r"(?i)^[\s*]*@jsxRuntime\s+(\S+)");
+    static JSX_FACTORY_RE: Lazy<regex::Regex> =
+      lazy_regex::lazy_regex!(r"(?i)^[\s*]*@jsxFactory\s+(\S+)");
+    static JSX_FRAGMENT_FACTORY_RE: Lazy<regex::Regex> =
+      lazy_regex::lazy_regex!(r"(?i)^[\s*]*@jsxFragmentFactory\s+(\S+)");
```

Comment:
I thought we already had these defined somewhere

Reply:
https://github.com/search?q=org%3Adenoland+jsxRuntime+language%3ARust+&type=code

Only for jsxImportSource/jsxImportTypes